mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -64,6 +64,12 @@ case class ScalaPresentationCompiler(
       config,
       sh,
       () => new Scala3CompilerWrapper(newDriver),
+      () => s"""|Scala version: $scalaVersion
+                |Classpath:
+                |${classpath.mkString(", ")}
```

Comment:
Don't you want to have that `missing`, `exists` info for classpath here too?

Reply:
good catchm added.